docs/tech/mvp/frameworks/dotnet.md

```diff
@@ -0,0 +1,20 @@
+# .NET
+
+## 👀 TLDR;
+
+[.NET](https://dot.net) is a framework for writing _any_ type of application, including, but not limited to, console apps, mobile apps (with MAUI) web APIs (with ASP.NET Core), games (with Unity3D) and machine learning models (with ML.NET).
```

Comment:
Zmienilbym Krzyśku `Unity3D` na `Unity`, jako ze taka jest nazwa silnika i mozna w nim tworzyc takze gry 2D. Dodalbym jeszcze `or MonoGame`. [MonoGame](https://www.monogame.net/) to aktywnie rozwijany projekt open-source'owy, w ktorym mozemy stworzyc wlasny silnik.

Reply:
Zmienilbym jeszcze `mobile apps (with MAUI)` na `mobile and desktop apps (with MAUI), ` (poza trescia dodalem jeszcze na koncu brakujacy przecinek).